### DISCUSSION TITLE
Jyers frimware for CR-10 Max

### DISCUSSION THREAD

**Discussion by Rekcams** [Jyers/Marlin#1052] [Q&A] [answered]
I love the new ender 3 v2 Jyers firmware update. Is there a CR-10 max version perhaps or will there be one in the future as I do like the Jyers on my ender 3 v2

**Comment by smooreace** [comment, marked as answer]
Its all about the display. Replace what you have with the correct display and you're most of the way there

**Reply by smooreace** [reply]
Then its all about the config changes, compile and prosper

**Reply by Rekcams** [reply]
Only problem is that the CR-10 is touchscreen, and its horizontal unlike the ender 3 v2 is vertical

**Reply by Cal-LF** [reply]
The touchscreen is not supported but I read there could be a horizontal option for the E3V2 screen in the works.
